scalableminds/webknossos
# Moving and deleting multiple trees in skeletontracing

This PR enables selecting multiple trees. Moving and deleting effect all selected trees.

### URL of deployed dev instance (used for testing):
- [https://movinganddeletingmultipletrees.webknossos.xyz](https://movinganddeletingmultipletrees.webknossos.xyz)

### Steps to test:
- open a skeletontracing
- create some trees and groups
- select some of them with ctrl + left mouse
- try deleting and moving the trees
- there should be no possibility to have a parent and a child selected simultaneously
- when deleting and groups are selected a modal should show up that asks whether to remove the groups and the subtrees or only the groups


### Issues:
- fixes #3143

------
- [x] Updated [documentation](../blob/master/docs) if applicable -> maybe
